clients/admin-ui/README.md

```diff
@@ -0,0 +1,28 @@
+# Admin UI
+
+Admin UI for managing FidesOps privacy requests. A web application built in Next.js with the FidesUI component library.
+
+## Running Locally
+
+1. Run `make server` in top-level `fidesops` directory, then run `make user` and follow prompts to create a user. Note that password requires 8 or more characters, upper and lowercase chars, a number, and a symbol.
```

Comment:
As per https://github.com/ethyca/fidesops/pull/373/files#r845571848 `make server` and `make user` can run independently of one another. `make user` must be run before the initial user logs in.